.github/pull_request_template.md

```diff
@@ -8,3 +8,4 @@
   - Any breaking changes to Swift or Kotlin binding APIs are noted explicitly
 - [ ] **Dependencies**: This PR follows our [dependency management guidelines](https://github.com/mozilla/application-services/blob/master/docs/dependency-management.md)
   - Any new dependencies are accompanied by a summary of the due dilligence applied in selecting them.
+- [ ] **Library Version** This PR makes a change that will require /libs to be rebuilt, go to /libs/ilbs-version, and change "UPDATED: FALSE" to "UPDATED: TRUE".
```

Comment:
This isn't sufficient. Like we discussed, there needs to be some versioning. If
it's a binary switch, who turns it off? Here's a recap:

1. There should be a file somewhere with a sequence of version numbers
   (corresponding to versions of the libraries). Initially this will be 0, but
   will increase whenever library versions need to be updated/rebuild.

   For example:
   ```
   2. Updated library bar.
   1. Updated library foo.
   0. Initial version.
   ```
1. There should be a file (e.g., `.curr_libs_version`) that contains the current
   version number of the built libraries. This isn't checked into version
   control (it should be in `.gitignore`) and is local to every developer.

   This file should contain the _current version_ of the libraries built
   locally.
1. When I build, this script should read the latest version from the sequence.
   It should then check for the existence of the `.curr_libs_version` file.
   * If that file doesn't exist, or the version number it contains is incorrect,
     your script should blow away the current libs and rebuild them. After it
     does, it should write that file with the current version.
   * If that file does exist, your script should do nothing.